pages/docs/user-docs/docs-recipes.md

```diff
@@ -2,60 +2,67 @@
 
+{% include toc.html %}
 
-## Quick Start
+## Overview
 
-Let's start with describing the file, globally. There are two kinds of components:
+A Singularity Recipe file is divided into several parts:
 
 1. **Header**: The Header describes the core operating system to build within the container. Here you will configure the base operating system features that you need within your container. Examples of this include, what distribution of Linux, what version, what packages must be part of a core install.
-2. **Sections**: The rest of the definition is comprised of sections, or blobs of data. Each section is defined by a `%` character followed by the name of the particular section. All sections are optional.
+2. **Sections**: The rest of the definition is comprised of sections, sometimes called scriptlets or blobs of data. Each section is defined by a `%` character followed by the name of the particular section. All sections are optional.
 
+Please see the [examples](https://github.com/singularityware/singularity/tree/master/examples) directory in the [Singularity source code](https://github.com/singularityware/singularity) for some ideas on how to get started. 
 
 ### Header
-The header is at the top of the file, and tells Singularity the base to use, and which one. Specifically:
+The header is at the top of the file, and tells Singularity the base Operating System that it should use to build the container.  It is composed of several keywords. Specifically:
 
- - **Bootstrap**: references the kind of base you want to use (e.g., docker, debootstrap, shub). For example, a shub bootstrap will pull containers for shub as bases. A Docker bootstrap will pull docker layers to start your image. For a full list see <a href="/build">build</a>
- - **From**: is the named container (shub) or reference to layers (Docker) that you want to use (e.g., vsoch/hello-world)
+ - `Bootstrap:` references the kind of base you want to use (e.g., docker, debootstrap, shub). For example, a shub bootstrap will pull containers for shub as bases. A Docker bootstrap will pull docker layers to start your image. For a full list see <a href="/build">build</a>
+ - `From:` is the named container (shub) or reference to layers (Docker) that you want to use (e.g., vsoch/hello-world)
 
+Depending on the value assigned to `Bootstrap:`, other keywords may also be valid in the header.
 
-For example, a very minimal Docker build might look like this:
+For example, a very minimal Singularity Hub build might look like this:
 
-```bash
-Bootstrap: docker
-From: ubuntu:latest
+```
+Bootstrap: shub
+From: singularityhub:ubuntu
 ```
```

Comment:
vsoch/hello-world

Reply:
I'm not sold on that.  What's the point of having the singularityhub user with the "official" containers if you don't encourage users to use them?

Reply:
I don't actually think it's useful. They aren't being used.